enterprise/app/filter/filter.tsx

```diff
@@ -49,43 +68,88 @@ export default class FilterComponent extends React.Component<FilterProps, State>
     const startDate = (startDateParam ? moment(startDateParam) : moment().subtract(7, "days")).toDate();
     const endDate = (endDateParam ? moment(endDateParam) : moment()).toDate();
 
-    const isFiltering = Boolean(this.props.search.toString());
+    const canClearFilters = Boolean(this.props.search.toString());
+
+    const roleValue = this.props.search.get(ROLE_PARAM_NAME) || "";
 
     return (
-      <div className={`global-filter container ${isFiltering ? "is-filtering" : ""}`}>
-        {isFiltering ? (
-          <OutlinedButton
-            className="clear-filters-button"
-            title="Clear filters"
-            onClick={this.onClickClearFilters.bind(this)}>
-            <img src="/image/x.svg" alt="" />
-          </OutlinedButton>
-        ) : (
-          <div className="filter-icon-container">
-            <img src="/image/filter.svg" alt="" />
-          </div>
+      <div className={`global-filter ${canClearFilters ? "is-filtering" : ""}`}>
+        {canClearFilters && (
+          <FilledButton className="square" title="Clear filters" onClick={this.onClickClearFilters.bind(this)}>
```

Comment:
Should we just show this if a filter is applied - not a date range?

Reply:
OK, sounds good -- maybe we can have a dedicated "Default" time range in the datepicker instead, and maybe make the text a different color when a non-default range is selected, or something.

Reply:
Done